nio_cli/commands/config.py

```diff
@@ -140,25 +141,42 @@ def success_message(name, ssl, self_signed, niohost, nioport, standalone_pubkeep
+    print('First, start your instance:')
+    print('- Enter your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m')
 
     if (standalone_pubkeeper and nioport <= 1024):
-        print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'sudo niod -s nio.conf -s pk_server.conf' + '\033[0m')
+        print('- Start the nio daemon: '  + '\033[94m' + 'sudo niod -s nio.conf -s pk_server.conf' + '\033[0m')
+        print('- Start the nio daemon inn the background: '  + '\033[94m' + 'sudo nohup niod -s nio.conf -s pk_server.conf 2>&1 > /dev/null &' + '\033[0m')
+
     elif (standalone_pubkeeper):
-        print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'niod -s nio.conf -s pk_server.conf' + '\033[0m')
+        print('- Start the nio daemon: '  + '\033[94m' + 'niod -s nio.conf -s pk_server.conf' + '\033[0m')
+        print('- Start the nio daemon in the background: '  + '\033[94m' + 'nohup niod -s nio.conf -s pk_server.conf 2>&1 > /dev/null &' + '\033[0m')
+
     elif (nioport <= 1024):
-        print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'sudo niod' + '\033[0m')
+        print('- Start the nio daemon: '  + '\033[94m' + 'sudo niod' + '\033[0m')
+        print('- Start the nio daemon in the background: '  + '\033[94m' + 'sudo nohup niod 2>&1 > /dev/null &' + '\033[0m')
+
     else:
-        print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'niod' + '\033[0m')
+        print('- Start the nio daemon: '  + '\033[94m' + 'niod' + '\033[0m')
+        print('- Start the nio daemon in the background: '  + '\033[94m' + 'nohup niod 2>&1 > /dev/null &' + '\033[0m')
 
     if (self_signed == True):
-        print('Next, accept your self-signed certificate by visiting ' + '\033[94m' + 'https://' + niohost + ':' + str(nioport) + '\033[0m' + ' and clicking "Advanced > Proceed to Site Anyway".' + '\033[0m')
+        print('')
+        print('Next, accept your self-signed certificate:')
+        print('- Visit ' + '\033[94m' + 'https://' + niohost + ':' + str(nioport) + '\033[0m')
+        print('- Click "Advanced > Proceed to Site Anyway".' + '\033[0m')
+
+    print('')
+    print('Then, add your local instance to the designer:')
 
     if (ssl == True):
-        print('Then, proceed to ' + '\033[94m' + 'https://app.n.io/design' + '\033[0m' + ' and add your local instance to the designer: ')
+        print('- Visit ' + '\033[94m' + 'https://app.n.io/design' + '\033[0m')
+        print('- Choose your system and click "Add Instance"')
         print('- hostname:' + '\033[94m' + ' https://' + niohost + '\033[0m')
```

Comment:
Isn't it the case that the designer can't handle `http` or `https` in front of hostnames? We may want to "time" this merge with that fix if so.

Reply:
It is the case... but to your point the other day, it doesn't work now, either... and worsely.

Reply:
I know that fix was being pulled forward, so I'm comfortable either way... your call.